### DISCUSSION TITLE
Properties empty []

### DISCUSSION THREAD

**Discussion by avisentinlektor** [spatie/laravel-activitylog#1274] [Q&A]
"spatie/laravel-activitylog": "^4.5.2"
"php": "8.0.7"

After the update, in the table of acitivity_log the properties are always an empty array.
id;log_name;description;subject_id;subject_type;causer_id;causer_type;properties;created_at;updated_at;batch_uuid;event
3937584;default;updated;6289344;App\Models\Eloquent\Sede\Lettura;24;App\Models\Eloquent\System\User;[];2024-02-06 17:08:53;2024-02-06 17:08:53;\N;updated

How i can fix?

**Comment by npalmqvist** [comment]
Having the same issue. 
Using: 
spatie/laravel-activitylog": "^4.10"
PHP 8.3 & Laravel 12

```
public function getActivitylogOptions(): LogOptions
{
    return LogOptions::defaults()
        ->logOnly(['*'])
        ->logOnlyDirty()
        ->dontSubmitEmptyLogs()
        ->dontLogIfAttributesChangedOnly(['updated_at'])
        ->useLogName('system');
}
```
But only get empty properties array.
